blocks/hero/hero.js

```diff
@@ -0,0 +1,15 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  const pic = block.querySelector('picture');
+  if (pic) {
+    const img = pic.querySelector('img');
+    const optimizedPicture = createOptimizedPicture(
+      img.src,
+      img.alt,
+      false,
+      [{ width: '750' }],
```

Comment:
Let's add some more breakpoints here, so we have appropriate image sizes for desktop, mobile etc.  For example 
const breakpoints = [{ media: '(min-width: 1000px)', width: '2000' }, { media: '(min-width: 500px)', width: '750' }, { width: '300' }];

Reply:
Done.